src/chaplin.coffee

```diff
@@ -1,23 +1,43 @@
 'use strict'
 
+import Application from './chaplin/application.coffee'
+import Composer from './chaplin/composer.coffee'
+import Controller from './chaplin/controllers/controller.coffee'
+import Dispatcher from './chaplin/dispatcher.coffee'
+import Composition from './chaplin/lib/composition.coffee'
+import EventBroker from './chaplin/lib/event_broker.coffee'
+import History from './chaplin/lib/history.coffee'
+import Route from './chaplin/lib/route.coffee'
+import Router from './chaplin/lib/router.coffee'
+import support from './chaplin/lib/support.coffee'
+import SyncMachine from './chaplin/lib/sync_machine.coffee'
+import utils from './chaplin/lib/utils.coffee'
+import mediator from './chaplin/mediator.coffee'
+import Collection from './chaplin/models/collection.coffee'
+import Model from './chaplin/models/model.coffee'
+import CollectionView from './chaplin/views/collection_view.coffee'
+import Layout from './chaplin/views/layout.coffee'
+import View from './chaplin/views/view.coffee'
+
 # Main entry point into Chaplin module.
 # Load all components and expose them.
-module.exports =
-  Application:    require './chaplin/application'
-  Composer:       require './chaplin/composer'
-  Controller:     require './chaplin/controllers/controller'
-  Dispatcher:     require './chaplin/dispatcher'
-  Composition:    require './chaplin/lib/composition'
-  EventBroker:    require './chaplin/lib/event_broker'
-  History:        require './chaplin/lib/history'
-  Route:          require './chaplin/lib/route'
-  Router:         require './chaplin/lib/router'
-  support:        require './chaplin/lib/support'
-  SyncMachine:    require './chaplin/lib/sync_machine'
-  utils:          require './chaplin/lib/utils'
-  mediator:       require './chaplin/mediator'
-  Collection:     require './chaplin/models/collection'
-  Model:          require './chaplin/models/model'
-  CollectionView: require './chaplin/views/collection_view'
-  Layout:         require './chaplin/views/layout'
-  View:           require './chaplin/views/view'
+export default {
+  Application
+  Composer
+  Controller
+  Dispatcher
+  Composition
+  EventBroker
+  History
+  Route
+  Router
+  support
+  SyncMachine
+  utils
+  mediator
+  Collection
+  Model
+  CollectionView
+  Layout
+  View
+}
```

Comment:
ES6 modules can do `export { foo } from 'bar.coffee'`, but that won't help here. Furthermore, we should be careful not break CommonJS users because of `default` thing.